packages/@uppy/companion/src/server/b2.js

```diff
@@ -0,0 +1,170 @@
+const request = require('request')
+
+const B2_API_VERSION = 2
+const B2_API_URL = `https://api.backblazeb2.com/b2api/v${B2_API_VERSION}/`
+
+module.exports = class B2Client {
```

Comment:
I have no particularly strong reason for avoiding the official module, other than the frustrating experience I've had trying to get axios to work in cloud worker environments, and their REST API is very well documented (IMO) and easy to work with directly. But swapping it out should be relatively easy if that would in fact be preferred.

Reply:
I have never tried using Axios in node environment (I'm also not sure what `cloud worker environments` mean here), but if the library is compatible in node environments then I'll be more in favor of using the npm library. This will relieve us of the weight of maintaining the client (for example if `backblazeb2` API gets updated)